rusk-recovery/config/genesis.toml

```diff
@@ -1,3 +0,0 @@
-[acl.stake]
```

Comment:
Is this file still necessary? (it's currently empty)

Reply:
Removed